### ISSUE TITLE
Name of Property Loss window

### ISSUE THREAD

**Issue by prispens** [joshwhitk/suppress-windows-copy-loss-properties#1]
Dear Josh, thanks for your script to suppress the most annoying question I've encountered so far. We copy like a hundred files a day from one map to another as part of archiving scanned documents to folders of our clients on a NAS (MyCloud by WD, which is an Unix machine with a Samba server). This is done by hand - there is no other way yet - and it means we have to click 'Yes' at least once per document to answer the question about copying without property loss.  One click isn't a problem, a hundred is already a lot of clicks and worth getting rid of when your answer will be 'Yes' all the time. I've installed the script, it runs, but it doesn't do the job. One thing is that in The Netherlands (were I live) the question in the pop-up window requires an answer in Dutch. So 'Yes' is 'Ja' and alt-y becomes alt-j. I've altered the script accordingly. Still not working. Can it be that the window the script is waiting for to appear ('Property Loss') can have a different name as well? And if so, how can I determine how the window is called? Or, do know any other cause of the script now working? Thanks so much in advance for your reply.

Paul, Leersum, NL

**Comment by joshwhitk** [comment]
Yes, I bet you're right that the window title is different in Dutch.

  To find the correct title on your computer, use
www.autoitscript.com/autoit3/docs/intro/au3spy.htm to find "window title"
and use that instead if the English string I hard coded.

Also, this code wastes CPU and memory when idle.  I am guessing the do
while loop is the culprit.  If you can be bothered updating it to be event
driven , it will be great.  Otherwise, just turn it off when you're not
using it.

Thanks for updating,

Josh

**Comment by prispens** [comment]
Thanks for you immediate reply Josh!
Meanwhile I've changed a few in the configuration here which lead to the
problem being solved: I moved the folder with the scanned items to the NAS,
eureka!
Still it would be great of it became a settings options in folder behavior
within Windows.
Thanks again, you put my thoughts on track on this!

Best!
Paul

2016-07-29 3:44 GMT+02:00 joshwhitk notifications@github.com:

> Yes, I bet you're right that the window title is different in Dutch.
> 
> To find the correct title on your computer, use
> www.autoitscript.com/autoit3/docs/intro/au3spy.htm to find "window title"
> and use that instead if the English string I hard coded.
> 
> Also, this code wastes CPU and memory when idle. I am guessing the do
> while loop is the culprit. If you can be bothered updating it to be event
> driven , it will be great. Otherwise, just turn it off when you're not
> using it.
> 
> Thanks for updating,
> 
> Josh
>
